Documentation/community/governance/commit_access.rst

```diff
@@ -181,7 +181,9 @@ The process to grant commit access to a candidate is simple:
    to the repos. The new committer is invited to the #committers Slack channel,
    *after* the nomination poll and related discussions have been deleted. The
    name of the new committer is also added to the list in the MAINTAINERS.md_
-   file.
+   file. The new committer is also added to the CNCF's list of Cilium
+   maintainers as documented `here
+   <https://github.com/cncf/foundation/README.md>`__. 
```

Comment:
This seems like a dead link to me?

Reply:
https://github.com/cncf/foundation#other-content (or https://github.com/cncf/foundation/blob/main/README.md#other-content)?